tagrecommendation/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM debian:10
+
+MAINTAINER Freesound support@freesound.org
+
+ENV PYTHONUNBUFFERED=1
+
+# Debian 10 installs python 2.7 as the 'python' package.
+# Double-check this if upgrading to a new version of debian
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+                       build-essential \
+                       ca-certificates \
+                       git \
+                       gosu \
+                       pkg-config \
+                       python \
+                       python-dev \
+                       wget \
+    && rm -rf /var/lib/apt/lists/*
+
+# pip
+ENV PYTHON_PIP_VERSION 20.0.2
+RUN set -ex; \
+	\
+	wget -O get-pip.py 'https://bootstrap.pypa.io/get-pip.py'; \
+	\
+	python get-pip.py \
+		--disable-pip-version-check \
+		--no-cache-dir \
+		"pip==$PYTHON_PIP_VERSION" \
+	; \
+	pip --version; \
+	\
+	find /usr/local -depth \
+		\( \
+			\( -type d -a \( -name test -o -name tests \) \) \
+			-o \
+			\( -type f -a \( -name '*.pyc' -o -name '*.pyo' \) \) \
+		\) -exec rm -rf '{}' +; \
+	rm -f get-pip.py
```

Comment:
If we are duplicating this in all of our dockerfiles, perhaps we should have a base dockerfile which does:
 - sets PYTHONUNBUFFERED
 - installs base packages
 - installs pip
 - creates fsweb user
 - adds entrypoint file